index.js

```diff
@@ -16,7 +16,12 @@ module.exports = function (content) {
   }
 
   if (config.rewritePath) {
-    const filePath = JSON.stringify(path.join(config.rewritePath, fileName));
+    let filePath;
+    if (config.rewritePath === './') {
```

Comment:
@farfromrefug Okay, please support Windows as well by extending the condition (probably something like `=== './' || === .\`) and it should be good to merge. Also, it would be good to write why this is needed in the commit message.